src/bot/events/interaction-create/embed/role-grant/create/validators/index.ts

```diff
@@ -0,0 +1,74 @@
+import { ChannelType, Guild, GuildMember, Interaction, PermissionFlagsBits, Role, TextChannel } from "discord.js"
+import { EmbedRoleGrantModalError } from "../handlers/modal"
+import { ERR, MSG } from "../messages"
+import { EmbedRoleGrantButtonError } from "../handlers/button"
+import { memberHasRole } from "../../../../../../../utils/discord"
+import { parseHexColor } from "../../../../../../../utils/color"
+import { decodeSnowflake } from "../../../../../../../utils/component"
```

Comment:
holy shit wth is this import path dawg 💀️️️️️️

https://bun.com/guides/runtime/tsconfig-paths